solutions/devsprint-riccardo-1/OnboardingChallenge/Screens/Settings/SettingsInteractor.swift

```diff
@@ -1,28 +1,21 @@
-//
-//  SettingsInteractor.swift
-//  OnboardingChallenge
-//
-//  Created by Strawberry Pie on 12/14/21.
-//
+// Copyright © 2021 Bending Spoons S.p.A. All rights reserved.
 
 import Foundation
 
-protocol SettingsBusinessLogic {
-    func getAppVersion(request: SettingsViewConfiguration.Get.Request)
+protocol LoadingPropertyInfo {
+  func getAppVersion(completion: @escaping (SettingsViewModel) -> Void)
```

Comment:
Why can't we return a `SettingsViewModel` directly instead of using a closure? `BundleService.appVersion` returns immediately, so there is no need to have a callback.